nikola/plugin_categories.py

```diff
@@ -644,18 +667,14 @@ def get_classification_friendly_name(self, classification, lang, only_last_compo
         The result of this function is usually displayed to the user, instead
         of using the classification string.
 
-        For hierarchical taxonomies, the result of extract_hierarchy is provided
-        as `classification`. For non-hierarchical taxonomies, the classification
-        string itself is provided as `classification`.
-
         The argument `only_last_component` is only relevant to hierarchical
         taxonomies. If it is set, the printable name should only describe the
         last component of `classification` if possible.
         """
         raise NotImplementedError()
 
-    def get_list_path(self, lang, type='page'):
-        """A path handler for the list of all classifications.
+    def get_overview_path(self, lang, type='page'):
+        """A path handler for the classification overview.
```

Comment:
Yeah, I noticed that I used two different terms for the same things and tried to adjust it all to "classification overview".